mitodl/open-api-clients
# Setup JS package

### What are the relevant tickets?




Closes #3 

### Description (What does it do?)
This adds the configuration necessary to compile the library to commonjs and package it as a library.

### How can this be tested?

- Run `yarn pack --dry-run` and verify that the output includes `dist/`, `LICENSE`, and `package.json`
